MetasysServices/MetasysClient.cs

```diff
@@ -473,6 +473,17 @@ public async Task<IEnumerable<MetasysObject>> GetNetworkDevicesAsync(string type
             return await NetworkDevices.GetAsync(type);
         }
 
+        /// <inheritdoc/>
+        public IEnumerable<MetasysObject> GetNetworkDevicesByClassification(string classification = null)

```

Comment:
Actually I would deprecate the methods related to the Network Devices directly defined in the 'MetasysClient'.
(I would add it to the 'NetworkDevices' service)

Reply:
Removed method GetNetworkDevicesByClassification from 'MetasysClient' and added to 'NetworkDevices' service